planetiler-core/src/main/java/com/onthegomap/planetiler/config/PlanetilerConfig.java

```diff
@@ -99,6 +101,8 @@ public static PlanetilerConfig from(Arguments arguments) {
       arguments.getDuration("loginterval", "time between logs", "10s"),
       arguments.getInteger("minzoom", "minimum zoom level", MIN_MINZOOM),
       arguments.getInteger("maxzoom", "maximum zoom level (limit 14)", MAX_MAXZOOM),
+      arguments.getInteger("minzoomForRendering", "minimum zoom level", MIN_MINZOOM),
+      arguments.getInteger("maxzoomForRendering", "maximum rendering zoom level (limit 14)", MAX_MAXZOOM),
```

Comment:
For naming, I think `maxzoom_for_render` or `render_maxzoom` (let's stick with lower_camel_case) makes sense, but maybe add to the description that minzoom/maxzoom are "minimum/maximum tile zoom level to generate" and something in the render max zoom that it's the zoom level that contains fine details for overzooming?

On a side note I almost wonder if it is more a property of the profile? Maybe we can revisit as we get deeper into config-driven schemas.